ls1intum/ArTEMiS
# Feature/lectures

### Checklist

### Motivation and Context
Lectures are required to distribute lecture material to the students.

### Description
This adds lectures to ArTEMiS. Lectures can be created and are visible in the student dashboard. for each Lecture attachments can be added. Those will be downloadable by the students

### Steps for Testing

1. Log in to ArTEMiS
2. Navigate to Course Administration
3. Select Lectures for a course
4. Create a lecture
5. create a attachment
6. check student dashboard if everything is displayed correctly

### Screenshots
![image](https://user-images.githubusercontent.com/10998002/56291720-16c26180-6126-11e9-87db-c1596180fcf4.png)
![image](https://user-images.githubusercontent.com/10998002/56291733-1e820600-6126-11e9-9f06-011c9978e82c.png)
![image](https://user-images.githubusercontent.com/10998002/56291767-29d53180-6126-11e9-8153-b8b49713446a.png)
![image](https://user-images.githubusercontent.com/10998002/56291787-348fc680-6126-11e9-95b6-2725f2e88a57.png)
